### ISSUE TITLE
Investigate possible issue with Snyk/Dependabot

### ISSUE THREAD

**Issue by jonasbn** [jonasbn/docker-cheatset#7]
> Workflows triggered by Dependabot on the "push" event run with read-only access. Uploading Code Scanning results 
> requires write access. To use Code Scanning with Dependabot, please ensure you are using the "pull_request" event for 
> this workflow and avoid triggering on the "push" event for Dependabot branches. See 
> https://docs.github.com/en/code-security/secure-coding/configuring-code-scanning#scanning-on-push for more 
> information on how to configure these events.

REF: https://github.com/jonasbn/docker-cheatset/actions/runs/760407030

**Comment by jonasbn** [comment]
This seem to have been fixed with commit ab924823b24e31fcbc2cd48d15d1f879c6900813

**Comment by jonasbn** [comment]
This issue was not fixed

```
Run github/codeql-action/upload-sarif@v1
  with:
    sarif_file: snyk.sarif
    checkout_path: /home/runner/work/docker-cheatset/docker-cheatset
    token: ***
    matrix: null
Error: Path does not exist: snyk.sarif
Error: Path does not exist: snyk.sarif
    at getSarifFilePaths (/home/runner/work/_actions/github/codeql-action/v1/lib/upload-lib.js:103:15)
    at Object.uploadFromActions (/home/runner/work/_actions/github/codeql-action/v1/lib/upload-lib.js:91:30)
    at run (/home/runner/work/_actions/github/codeql-action/v1/lib/upload-sarif-action.js:34:46)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async runWrapper (/home/runner/work/_actions/github/codeql-action/v1/lib/upload-sarif-action.js:46:9)
```

REF: https://github.com/jonasbn/docker-cheatset/runs/2375482599?check_suite_focus=true
